src/DGtal/io/Display3DFactory.h

```diff
@@ -613,7 +613,7 @@ namespace DGtal
      * @param name3d an object storing the "OpenGL" name.
      */
     static void 
-    draw( Display & display, const DGtal::SetName3D& name3d );
+    draw( Display & display, const DGtal::SetName3D& aName3d );
```

Comment:
Same comment as above. The comment above should be update "name3d" => "aName3d'.